src/Sulu/Bundle/MediaBundle/Entity/MediaDataProviderRepository.php

```diff
@@ -168,7 +168,7 @@ protected function appendTagsRelation(QueryBuilder $queryBuilder, $alias)
     /**
      * {@inheritdoc}
      */
-    protected function appendCategoriesRelation()
+    protected function appendCategoriesRelation(QueryBuilder $queryBuilder, $alias)
```

Comment:
The `DataProviderRepositoryInterface` which is implement define this function with the given parameters.